shelley/chain-and-ledger/formal-spec/chain.tex

```diff
@@ -333,6 +333,22 @@ \subsection{New Epoch Transition}
 \end{itemize}
 
 \begin{figure}[ht]
+  \begin{equation}\label{eq:no-reward-update}
+    \inference[No-Reward-Update]
+    {
+      \var{ru} = \Nothing
```

Comment:
Should we have `\var{ru} \neq \Nothing` as requirement for `New-Epoch` then?

Reply:
so that the rules are not mutually exclusive, and do not depend on ordering?

Reply:
Yes, I think that would be good. But I am not sure we do this everywhere else.

Reply:
ok, it' all mutually exclusive now, and does exactly what the exec model does.